_data/155.yml

```diff
@@ -113,52 +113,64 @@ downloads:
+        url: /distribution/leap/15.5/appliances/openSUSE-Leap-15.5-Minimal-VM.x86_64-kvm-and-xen.qcow2.meta4
       - name: pick_mirror
-        url: /distribution/leap/15.5/appliances/openSUSE-Leap-15.5-JeOS.x86_64-kvm-and-xen.qcow2?mirrorlist
+        url: /distribution/leap/15.5/appliances/openSUSE-Leap-15.5-Minimal-VM.x86_64-kvm-and-xen.qcow2?mirrorlist
       - name: checksum
-        url: /distribution/leap/15.5/appliances/openSUSE-Leap-15.5-JeOS.x86_64-kvm-and-xen.qcow2.sha256
+        url: /distribution/leap/15.5/appliances/openSUSE-Leap-15.5-Minimal-VM.x86_64-kvm-and-xen.qcow2.sha256
     - name: hyperv_image
       short: hyperv_desc
-      primary_link: /distribution/leap/15.5/appliances/openSUSE-Leap-15.5-JeOS.x86_64-MS-HyperV.vhdx.xz
+      primary_link: /distribution/leap/15.5/appliances/openSUSE-Leap-15.5-Minimal-VM.x86_64-MS-HyperV.vhdx.xz
       links:
       - name: metalink
-        url: /distribution/leap/15.5/appliances/openSUSE-Leap-15.5-JeOS.x86_64-MS-HyperV.vhdx.xz.meta4
+        url: /distribution/leap/15.5/appliances/openSUSE-Leap-15.5-Minimal-VM.x86_64-MS-HyperV.vhdx.xz.meta4
       - name: pick_mirror
-        url: /distribution/leap/15.5/appliances/openSUSE-Leap-15.5-JeOS.x86_64-MS-HyperV.vhdx.xz?mirrorlist
+        url: /distribution/leap/15.5/appliances/openSUSE-Leap-15.5-Minimal-VM.x86_64-MS-HyperV.vhdx.xz?mirrorlist
       - name: checksum
-        url: /distribution/leap/15.5/appliances/openSUSE-Leap-15.5-JeOS.x86_64-MS-HyperV.vhdx.xz.sha256
+        url: /distribution/leap/15.5/appliances/openSUSE-Leap-15.5-Minimal-VM.x86_64-MS-HyperV.vhdx.xz.sha256
     - name: vmware_image
       short: vmware_desc
-      primary_link: /distribution/leap/15.5/appliances/openSUSE-Leap-15.5-JeOS.x86_64-VMware.vmdk.xz
+      primary_link: /distribution/leap/15.5/appliances/openSUSE-Leap-15.5-Minimal-VM.x86_64-VMware.vmdk.xz
       links:
       - name: metalink
-        url: /distribution/leap/15.5/appliances/openSUSE-Leap-15.5-JeOS.x86_64-VMware.vmdk.xz.meta4
+        url: /distribution/leap/15.5/appliances/openSUSE-Leap-15.5-Minimal-VM.x86_64-VMware.vmdk.xz.meta4
       - name: pick_mirror
-        url: /distribution/leap/15.5/appliances/openSUSE-Leap-15.5-JeOS.x86_64-VMware.vmdk.xz?mirrorlist
+        url: /distribution/leap/15.5/appliances/openSUSE-Leap-15.5-Minimal-VM.x86_64-VMware.vmdk.xz?mirrorlist
       - name: checksum
-        url: /distribution/leap/15.5/appliances/openSUSE-Leap-15.5-JeOS.x86_64-VMware.vmdk.xz.sha256
-    - name: openstack_image
-      short: openstack_desc
-      primary_link: /distribution/leap/15.5/appliances/openSUSE-Leap-15.5-JeOS.x86_64-OpenStack-Cloud.qcow2
+        url: /distribution/leap/15.5/appliances/openSUSE-Leap-15.5-Minimal-VM.x86_64-VMware.vmdk.xz.sha256
+    - name: cloud_image
+      short: cloud_minimal_desc
+      primary_link: /distribution/leap/15.5/appliances/openSUSE-Leap-15.5-Minimal-VM.x86_64-OpenStack-Cloud.qcow2
```

Comment:
New url would be /distribution/leap/15.5/appliances/openSUSE-Leap-15.5-Minimal-VM.x86_64-Cloud-Build7.76.qcow2